activity_calendar/forms.py

```diff
@@ -288,3 +291,15 @@ def save(self, commit=True):
             slot_obj.participants.add(self.user)
 
         return slot_obj
+
+
+class ActivityMomentForm(ModelForm):
```

Comment:
This creates an instance in the DB upon saving even if all (relevant) fields are left empty.
I'm also not sure if we want to delete an already existing activityMoments in case the user saves when all fields are empty.

Reply:
In the long term I think we want to link all slots an activity moment instance. So having an empty activityMoment instance is not a bad thing. In all relevant views we already initialise an activitymoment, it's just not present on the database.